.github/workflows/build_test.sh

```diff
@@ -130,17 +130,22 @@ ninja --version
 
 for args in "${ARGS[@]}"; do
     SECONDS=0
+    FCOMMON="-Werror"
```

Comment:
This is not right. It'll make things fail whenever any warning is emitted, making CI very flakey.

Reply:
I cannot follow. The flag has been already set unconditionally.

Reply:
Yeah, we already have `--werror` in the meson call below, so this should be dropped.